UI_UX/Frontend/src/components/Achievements.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import '../styles/Achievements.css';
+
+export class Achievements extends Component {
+  constructor(props) {
+    super(props);
```

Comment:
This is an unnecessary constructor, no need to set it up if you are not binding any event or using the state. You can just remove the `constructor` statement altogether.